### DISCUSSION TITLE
Bond as penalty mechanism and incentive for good behavior

### DISCUSSION THREAD

**Discussion by nobu-maeda** [MostroP2P/mostro#54] [Ideas]
Another question for whether this is a good idea or not. Both Bisq and RoboSat uses this extensively. Bisq in a more sophisticated manner to avoid needing a custodial coordinator in the middle of the trade.

**Comment by grunch** [comment]
This topic has been discussed for quite some time in Latin American forums and chats and after almost two years of live of [lnp2pBot](https://github.com/lnp2pBot/bot) we started to discuss again about it.

Our conclusion is bonds are not going to avoid scams, the real scams we face now is the triangulation where a scammer buys using the account of a hacked account, the scammer send the fiat money and the seller release funds, everything looks normal to seller, few days later the real owner of the hacked account call to the bank and they block and reverse the operation, the sellers is screwed, this can't be avoided with bonds.

Then bonds add more friction to new users, making impossible new users to buy because they don't have sats to pay the bond, for this we don't think bond could help.

**Reply by nobu-maeda** [reply]
Yeah, chargeback is an issue. Only reputation system of some sort might help with that.

**Reply by grunch** [reply]
We can't avoid that this happens but with a reputation system we can at least make it more difficult to the scammer

**Reply by nobu-maeda** [reply]
Agree. The only challenge is with reputations in P2P systems itself.

Bonds do encourage forward progress in trade and spam prevention however. Does not prevent chargebacks.

**Reply by grunch** [reply]
I think for spam prevention we will support NIP 13

**Reply by nobu-maeda** [reply]
Depending on how one defines SPAM, PoW probably insufficient to deter a bunch of categories of SPAM

**Reply by grunch** [reply]
Let's see if it works, a lot of people told me that using hold invoices wasn't a good idea but I tried it and it worked, we are experimenting on most of this things
